TECHNICAL_CHARTER.md

```diff
@@ -10,13 +10,13 @@ This charter (the "Charter") sets forth the responsibilities and procedures for
 
     b. The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source project.
 
-2. **Technical Steering Committee**
+2. **Community Governance Committee**
 
-    a. The Technical Steering Committee (the "TSC") will be responsible for all technical oversight of the open source Project.
+    a. The Community Governance Committee (the "CGC") will be responsible for all technical oversight of the open source Project.
```

Comment:
This isn't really the current mandate of the CGC, although it's true in a limited fashion for issues that can't be resolved in SIG Arch if I remember correctly. A more accurate description would be "community and process oversight" I think.

Reply:
Another way to say this is that the CGC is responsible for technical oversight, and has chosen to delegate it to SIG-Arch.

I believe that is well within the power and intent of the original Technical Charter as updated here.

Reply:
I think it may be helpful to phrase qualify this, because historical definition (https://github.com/openenclave/openenclave/commit/3e618d324fc4fc0beec2ba786107388dd39210c8#diff-e4cf1e1a589105199a1f764df35082a7) and current practice for the CGC is that it strives not to make technical decisions as much as possible. I understand the intention here, but I worry a first-time reader may get the impression that the CGC is in fact a technical committee (or even the technical committee).

Could this perhaps say "ultimately responsible", or "is responsible etc, but delegates technical oversight in various areas of the project to the relevant SIGs"?

I'm happy for this to stay as is if this is just a personal misunderstanding of course.

Reply:
I want to second Amaury's recommendation for phrasing - "ultimately responsible", or "is responsible etc, but delegates technical oversight in various areas of the project to the relevant SIGs"